### DISCUSSION TITLE
把同步到本地的文档删了，再执行npm run elog:sync-local，提示：没有需要同步的文档。我该如何再次下载语雀的文档

### DISCUSSION THREAD

**Discussion by SuperYIP** [LetTTGACO/elog#50] [Q&A] [answered]
把同步到本地的文档删了，再执行npm run elog:sync-local，提示：没有需要同步的文档。我该如何再次下载语雀的文档

**Comment by LetTTGACO** [comment, marked as answer]
请查看文档：
[如何重新全量同步文档](https://elog.1874.cool/notion/qa#%E5%A6%82%E4%BD%95%E9%87%8D%E6%96%B0%E5%85%A8%E9%87%8F%E5%90%8C%E6%AD%A5%E6%96%87%E6%A1%A3)
